src/components/ReportActionItem/IOUAction.js

```diff
@@ -43,29 +46,34 @@ const IOUAction = (props) => {
     const launchDetailsModal = () => {
         Navigation.navigate(ROUTES.getIouDetailsRoute(props.chatReportID, props.action.originalMessage.IOUReportID));
     };
+
+    const shouldShowIOUPreview = (
+        props.isMostRecentIOUReportAction
+        && Boolean(props.action.originalMessage.IOUReportID)
+        && props.chatReport.hasOutstandingIOU) || props.action.originalMessage.type === 'pay';
```

Comment:
I _think_ `props.chatReport.hasOutstandingIOU || props.action.originalMessage.type === 'pay'` alone might be enough.

Reply:
@youssef-lr That will make the IOUPreview shown on every IOU report.
<img width="427" alt="Screen Shot 2022-12-08 at 12 47 51" src="https://user-images.githubusercontent.com/25520267/206367648-3d71d6b2-316f-47e0-be24-49bd896dd632.png">

We should keep `props.isMostRecentIOUReportAction` to show IOUPreview for recent IOUReport.
<img width="405" alt="Screen Shot 2022-12-08 at 12 51 18" src="https://user-images.githubusercontent.com/25520267/206368019-93a3b5b1-31b9-47e6-baff-9562291d0d6b.png">

Reply:
@youssef-lr 
> I think props.chatReport.hasOutstandingIOU || props.action.originalMessage.type === 'pay' alone might be enough.

this wont be enough will it, you can send money no matter if you have existing pay request there

Reply:
Yeah @mollfpr @mountiny sorry scratch that.